### ISSUE TITLE
Lay groundwork for distribution via npm and CDNs

### ISSUE THREAD

**Issue by erictheise** [Maps4HTML/Web-Map-Custom-Element#493]
It'd be useful for releases to be available via `npm` and, indeed, there was some movement in this direction in #138 when v0.8.0 was released last year. Lodging this as an issue and looking into this over the next few days.

**Comment by Malvoz** [comment]
Perhaps a bit too much to ask as part of this issue, but it'd be really neat if we - at some point - could serve assets over a CDN. Apart from the potential performance benefits we could also use absolute URLs in our [HTML markup examples](https://maps4html.org/web-map-doc/docs/maps/mapml-viewer#introduction), which would provide developers with a third easy-to-use [installation alternative](https://maps4html.org/web-map-doc/docs/installation), as it is simply a matter of copy and paste. I have no personal experience in this area, but this resource may be useful: https://unpkg.com/#workflow.

**Comment by erictheise** [comment]
I was able to publish 0.8.0 to npm. It's at https://www.npmjs.com/package/@maps4html/web-map-custom-element

One caveat is that to install it into a project using `15.11.0` I had to use the `--legacy-peer-deps` flag, which resulted in:

```
$ npm install --save @maps4html/web-map-custom-element --legacy-peer-deps
npm WARN deprecated deep-defaults@1.0.5: not actively maintained; find alternatives
npm WARN deprecated transformation-matrix-js@2.7.6: Package no longer supported. Contact support@npmjs.com for more info.
npm WARN deprecated rollup-plugin-babel@4.4.0: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-babel.
```

**Comment by erictheise** [comment]
In chat @prushforth notes "On my windows machine I have v13.13.0 and on wsl2 it is v15.11.0" and that "The LTS version available for windows 10 is 14.7.3 / npm 6.14.13".

I use [nvm](https://github.com/nvm-sh/nvm) and note that `14.7.0` is what they have available. I'd be keen to commit an `.nvmrc` file containing either `15.11.0` or `14.7` but wonder what @Malvoz thinks about agreeing on a version to keep `package-lock.json` from flapping around too much.

**Comment by erictheise** [comment]
> Perhaps a bit too much to ask as part of this issue, but it'd be really neat if we - at some point - could serve assets over a CDN.

I think this is straightforward enough that we can piggyback it onto this issue.
